### PR TITLE
Shake sensor app

### PR DESCRIPTION
**Pull request by Kaliine**
My reflections on how this week's project turned out: It's still a working progress.

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Comment by Kaliine** [comment]
This project is not ready yet.
It's an app for children who are curious about space. Click the button or shake the phone to learn cool facts about space.

File: components/SensorComponent.js [hunk 1]
```diff
@@ -1,96 +1,122 @@
...
     y: 0,
     z: 0,
-  });
+  })
+
+  const [subscription, setSubscription] = useState(null)
 
-  // This keeps track of whether we are listening to the Accelerometer data
-  const [subscription, setSubscription] = useState(null);
+  useEffect(() => {
```
Comment (Jsfrulle) [inline components/SensorComponent.js:64]: Good naming of the elemts in your code! :D

File: components/SensorComponent.js [hunk 2]
```diff
@@ -1,96 +1,122 @@
...
+  const unsubscribe = () => {
+    subscription && subscription.remove()
+    setSubscription(null)
+  }
 
-    // Stop listening to the data when we leave SensorComponent
-    return () => _unsubscribe();
-  }, []);
+  const isShakingEnough = (spacefact) => {
+    const totalForce = Math.abs(spacefact.x) + Math.abs(spacefact.y) + Math.abs(spacefact.z)
```
Comment (Jsfrulle) [inline components/SensorComponent.js:88]: same here! Really discribing name!

File: components/SpaceFact.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+`
+const ButtonTopText = styled.Text`
+font-weight: 900;
+font-size: 18px;
+color: #fff;
+`
+const ButtonText = styled.Text`
+font-weight: 700;
+font-size: 14px;
+padding: 0 5px 0 5px;
```
Comment (Jsfrulle) [inline components/SpaceFact.js:116]: I think you have a really good foundation to complet this project. Your code is easy to follow and your nameing of the component is really good! I see that you have the same arry in two places,so just kepp one(I think you allready know that do, hehe :) )!Good work Caroline, Share the completed project when your done, i think is going to be awesome! :D

**Pull request closed without merging**